content/en/dashboards/guide/custom_time_frames.md

```diff
@@ -19,7 +19,7 @@ Both fixed and relative custom time frames are supported:
 
 {{< img src="dashboards/guide/custom_time_frames/custom_relative_time_frame.mp4" alt="Type custom relative time frame" video="true" width="500" >}}
 
-**Note**: Queries are run in UTC time, but the query time range is selected according to your browser. Additionally, you can toggle between displaying the default time zone or UTC from the [dashboard settings][1].
+**Note**: Queries are run in UTC time, but the query time range is selected according to your browser. Toggle between displaying the default time zone or UTC from the dashboard configure action. For more information, see the [Dashboard Settings][1] documentation.
```

Comment:
I would try to highlight (in bold for example) the fact that the timezone should be the user's browser.